tock/projects/models.py

```diff
@@ -247,6 +247,11 @@ class Project(models.Model):
         default=False,
         help_text='Check if this project should be excluded from calculations of billable hours, e.g. Out of Office'
     )
+    include_in_utilization = models.BooleanField(
+            default=False,
```

Comment:
For all existing Projects, we want to set the value of this to the project's billable status. This sets _all_ of them to `False`

Reply:
tactically, starting with the most common value and can update in separate PR

Reply:
I think there are a couple of different ways to handle this, but I tend to extend the auto-generated migration to set the defaults. Here's an example:

https://github.com/18F/tock/blob/1b8b88039d4b52e8c184281e392c1fde65c890c6/tock/hours/migrations/0055_auto_20200430_1742.py#L7-L40